source/developer/toolkit.rst

```diff
@@ -25,18 +25,16 @@ Below is a list of planned features of the developer toolkit with estimated time
 1. Completed:
 
  - Webhooks and slash commands to allow easy, low-effort extension and integration.
-
-2. Q2/2017:
-
  - Mattermost HTTP REST APIv4 allowing for much more powerful server interaction.
  - Mattermost Webapp moved over to Redux infrustructure.
 
-3. Q3/2017:
+2. Q3/2017:
 
- - Mattermost Webapp UI components moved over to be Redux compatible.
+ - API developer token to provide a simple method to authenticate to the Mattermost REST API.
  - The ability to build webapp client plugins to override existing UI components (replace posts with your custom components, use your own video services etc.), modify/extend client drivers to interact with custom server API endpoints, and add whole new UI views in predetermined places.
 
-4. Unscheduled:
+3. Unscheduled:
+ - Mattermost Webapp UI components moved over to be Redux compatible.
```

Comment:
Thanks, removed that item. There's already an item about moving the webapp to Redux in the completed section